### PR TITLE
SC-2330 Add vertical radiocard style

### PR DESCRIPTION
**Pull request by antonio-garcia-sonarsource**
With this vertical option we introduce a simple design with no minimum-height, full width and using a light border color on hover with no shadow or translation.

**Checklist**

* [x] Functional validation
* [x] Documentation update
* [x] Update changelog

### CODE REVIEW

File: components/controls/RadioCard.css
Comment (gregaubert) [inline components/controls/RadioCard.css]: Shouldn't it be `.radio-card-actionable.radio-card-vertical....` ? We never want the hover effect if the card is not actionable.

File: components/controls/RadioCard.css
Comment (gregaubert) [inline components/controls/RadioCard.css]: Same for this one.

File: components/controls/RadioCard.css
Comment (gregaubert) [inline components/controls/RadioCard.css:81]: It could be good to have comment explaining why we don't want to have the transform in the vertical form

**Review by gregaubert — APPROVED** [review]
Just a little fix to do on the css selectors, otherwise it looks good to me.
Let's have a SQ team member have a quick look too before merging, I'll ask review from @philippe-perrin-sonarsource :smile:

File: components/controls/RadioCard.tsx
Comment (philippe-perrin-sonarsource) [inline components/controls/RadioCard.tsx]: NP: This could be mixed with the line above.

File: CHANGELOG.md
Comment (philippe-perrin-sonarsource) [inline CHANGELOG.md]: Add the ticket number?

**Review by philippe-perrin-sonarsource — APPROVED** [review]
LGTM. I like how it looks.

Do you use the _non-vertical_ one somewhere in SonarCloud and if so, do you plan to adjust the styling for it too? I'm asking because this component is barely used on our side. We may move everything (SC/SQ) to the new styling.

**Comment by gregaubert** [comment]
> LGTM. I like how it looks.
> 
> Do you use the _non-vertical_ one somewhere in SonarCloud and if so, do you plan to adjust the styling for it too? I'm asking because this component is barely used on our side. We may move everything (SC/SQ) to the new styling.

We use it to suggest users to pass to a paid plan and somewhere in the members page, I think it make sense to update the styling for this version too indeed!
On SQ you use it on the new code configuration page, right ? Is it fine if we change a bit the style ?

**Comment by antonio-garcia-sonarsource** [comment]
Thanks for the review! I updated the PR with the new hover style and it works great for SC. So if it's good by SQ then we can get rid of the shadow+translate style entirely.

**Comment by philippe-perrin-sonarsource** [comment]
> Thanks for the review! I updated the PR with the new hover style and it works great for SC. So if it's good by SQ then we can get rid of the shadow+translate style entirely.

I've just pinged my teammates, we'll see. Go ahead with your changes if you can't/don't want to wait for their feedback. We can always proceed with this later.

**Comment by antonio-garcia-sonarsource** [comment]
Yes, I am going to go ahead with these changes, thanks!

**Pull request merged**
